x/epochs/keeper/hooks.go

```diff
@@ -5,11 +5,11 @@ import (
 )
 
 // AfterEpochEnd gets called at the end of the epoch, end of epoch is the timestamp of first block produced after epoch duration.
-func (k Keeper) AfterEpochEnd(ctx sdk.Context, identifier string, epochNumber int64) {
-	k.hooks.AfterEpochEnd(ctx, identifier, epochNumber)
+func (k Keeper) AfterEpochEnd(ctx sdk.Context, identifier string, epochNumber int64) error {
+	return k.hooks.AfterEpochEnd(ctx, identifier, epochNumber)
 }
 
 // BeforeEpochStart new epoch is next block of epoch end block
-func (k Keeper) BeforeEpochStart(ctx sdk.Context, identifier string, epochNumber int64) {
-	k.hooks.BeforeEpochStart(ctx, identifier, epochNumber)
```

Comment:
Can we ignore the error in here, instead of in abci.go?

Reply:
done!